packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -39,7 +54,13 @@ export default function SidebarNavigationScreen( {
 							) }
 						/>
 					) : (
-						<div className="edit-site-sidebar-navigation-screen__icon-placeholder" />
```

Comment:
I think there's some lingering styles that target this classname that we should remove

Reply:
Thanks, removed now.